### PR TITLE
EnvMon plugin

### PR DESCRIPTION
**Pull request by zodeak**

### CODE REVIEW

File: src/plugins/envmon/envmon.cpp [hunk 1]
```diff
@@ -0,0 +1,513 @@
+/*********************IMPORTANT DRAKVUF LICENSE TERMS***********************
+*                                                                         *
+* DRAKVUF (C) 2014-2017 Tamas K Lengyel.                                  *
```
Comment (tklengyel) [inline src/plugins/envmon/envmon.cpp]: 2019

File: src/plugins/envmon/envmon.h [hunk 2]
```diff
@@ -0,0 +1,146 @@
+/*********************IMPORTANT DRAKVUF LICENSE TERMS***********************
+*                                                                         *
+* DRAKVUF (C) 2014-2017 Tamas K Lengyel.                                  *
```
Comment (tklengyel) [inline src/plugins/envmon/envmon.h]: 2019

File: src/plugins/envmon/private.h [hunk 3]
```diff
@@ -0,0 +1,139 @@
+/*********************IMPORTANT DRAKVUF LICENSE TERMS***********************
+*                                                                         *
+* DRAKVUF (C) 2014-2017 Tamas K Lengyel.                                  *
```
Comment (tklengyel) [inline src/plugins/envmon/private.h]: 2019

File: src/plugins/plugins.h [hunk 4]
```diff
@@ -111,6 +111,22 @@
 #include <sys/time.h>
 #include <libdrakvuf/libdrakvuf.h>
 
+struct plugins_options
```
Comment (tklengyel) [inline src/plugins/plugins.h]: I like this!

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by tklengyel** [comment]
Please provide a description of the plugin in the commit message for this PR, thanks! Otherwise the PR looks mostly good to me

**Comment by ultrapikachu** [comment]
@zodeak plugin didn't give any output

**Pull request merged**